git.md

```diff
@@ -0,0 +1,35 @@
+## git
+
+### commit
+
+`git commit` saves changes
+
+Example:
+```console
+$ git commit -m "Enter here a message"
+```
+### status
+`git status`  Shows the state of working directory.
+```console
+$ git status
+```
+### push
+`git push` It lets you move a local branch to another repository.
+```console
+$ git push origin main
+```
+### add
+`git add` Moves changes from the working directory to the staging area.
+```console
+$ git add FROG.md
+```
+### diff 
+`git diff` Diff command is used in git to track the difference between the changes made on a file.
```

Comment:
Make it as a sentence: `git diff` shows difference ...

Reply:
fixed